content/terms/security.md

```diff
@@ -14,31 +14,30 @@ If you think you have discovered a security vulnerability in our product, please
+- All communication to your Sourcegraph instance is encrypted using TLS 1.2 or greater. User credentials are encrypted at rest using 256-bit Advanced Encryption Standard (AES-256) keys in Galois Counter Mode (GCM). The keys are automatically rotated every 90 days.
 - All storage volumes are encrypted at rest, and data is encrypted in-cloud during transport.
 - We leverage IAM groups and rules to enforce the principle of least access across our cloud infrastructure.
-- The domain sourcegraph.com is managed through Cloudflare and uses its security capabilities, like Web Application Firewall and Rate Limiting.
-- External access to production systems is restricted by firewall. Secrets that grant access to
-  compute resources are stored only on encrypted local drives or a secret management service.
-
-#### Monitoring and Incident Response
+- Domains are managed through Cloudflare and use its security capabilities, like Web Application Firewall and Rate Limiting.
+- External access to production systems is restricted by firewall. Secrets that grant access to compute resources are stored only on encrypted local drives or a secret management service.
+- Instances are updated monthly, and are actively maintained to keep the service up and healthy.
+- Instances are updated for security patches as needed, according to Sourcegraph's [Vulnerability Management Policy](https://handbook.sourcegraph.com/departments/engineering/dev/policies/vulnerability-management-policy/).
+  
+### Monitoring and Incident Response
 
-- Our operations team monitors service availability 24x7x365. They investigate alerts and potential attacks 24x7x365, triaging and responding if necessary.
-- We only log information crucial for security and support. Only restricted personnel have access to user data. Logs are stored in GCP and the information is retained for 180 days. Find out more in our [Privacy Policy](/terms/privacy).
-- Service, application, and access logs for sourcegraph.com are stored centrally by Sourcegraph and monitored.
-- Our Incident Response plan is currently under review. It will be publicly available when finalized.
+- Our operations team monitors service availability 24x7x365. They investigate alerts and potential attacks 24x7x365, triaging and responding when necessary.
+- We only log information crucial for security and support. Only restricted personnel have access to user data. Logs are stored in GCP and the information is retained for up to 180 days. Find out more in our [Privacy Policy](/terms/privacy).
+- Service, application, and access logs for are stored centrally by Sourcegraph and monitored.
+- You can find more details in our [Incident Response Policy](https://handbook.sourcegraph.com/departments/security/security-incident-response/)
+- Only restricted Sourcegraph employees have access to the instance, stricly for support and maintanace purposes. Access is heavily logged and monitored.
```

Comment:
* what is 'the instance' here?
* maintanance -> maintenance
* I would remove 'heavily' here, it's kind of vague